src/quo2/components/inputs/locked_input/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.inputs.locked-input.view
+  (:require [react-native.core :as rn]
+            [status-im2.contexts.quo-preview.tabs.segmented-tab]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]
```

Comment:
Use the icon component from the `quo2` namespace. `status-im` namespace contain outdated code.